sdk/core/azure-core/src/main/java/com/azure/core/credential/SimpleTokenCache.java

```diff
@@ -20,11 +20,21 @@
 public class SimpleTokenCache {
     // The delay after a refresh to attempt another token refresh
     private static final Duration REFRESH_DELAY = Duration.ofSeconds(30);
+    private static final String REFRESH_DELAY_STRING = String.valueOf(REFRESH_DELAY.getSeconds());
     // the offset before token expiry to attempt proactive token refresh
     private static final Duration REFRESH_OFFSET = Duration.ofMinutes(5);
     // SimpleTokenCache is commonly used, use a static logger.
     private static final ClientLogger LOGGER = new ClientLogger(SimpleTokenCache.class);
-    private final AtomicReference<Sinks.One<AccessToken>> wip;
+
+    private static final String NO_CACHE_ACQUIRED = "Acquired a new access token.";
+    private static final String NO_CACHE_FAILED = "Failed to acquire a new access token.";
+
+    private static final String NEGATIVE_TTE = " seconds after expiry. Retry may be attempted after "
+        + REFRESH_DELAY_STRING + " seconds.";
+    private static final String POSITIVE_TTE = " seconds before expiry. Retry may be attempted after "
+        + REFRESH_DELAY_STRING + " seconds. The token currently cached will be used.";
+
+    final AtomicReference<Sinks.One<AccessToken>> wip;
```

Comment:
Add a package-private getter method and keep `wip` private so it doesn't get modified outside the class.